plugins/modules/cloud/univention/udm_share.py

```diff
@@ -90,12 +90,6 @@
         description:
             - Blocking locks.
         aliases: [ sambaBlockingLocks ]
-    samba_browseable:
```

Comment:
This parameter exists, but there is a typo in the parameter. See [L#347](https://github.com/ansible-collections/community.general/blob/main/plugins/modules/cloud/univention/udm_share.py#L347), the parameter should be renamed to `samba_browseable`.

Reply:
I think the documentation should be fixed instead of the argument spec, to avoid a breaking change. (Also AFAIK `browsable` is correct English, while `browseable` is not.)

Reply:
Ok, that's a good point.